src/plugins/camera_tracking/CameraTracking.cc

```diff
@@ -187,13 +206,26 @@ void CameraTrackingPrivate::Initialize()
   gzmsg << "Move to service on ["
          << this->moveToService << "]" << std::endl;
 
-  // follow
   this->followService = "/gui/follow";
   this->node.Advertise(this->followService,
       &CameraTrackingPrivate::OnFollow, this);
   gzmsg << "Follow service on ["
          << this->followService << "]" << std::endl;
 
+  // track
+  this->trackTopic = "/gui/track";
```

Comment:
nit: how about `/gui/camera/track`? This would be a lot more clear immediately without needing to look into more detail

Reply:
then again, the other services `move` services don't use this format 😢 so I won't block on it

Reply:
Ok, so you do, or do not want it changed?

Reply:
I went without something like the camera prefix as I viewed it integral to having the gui do any for of "tracking" much like "move to".

Reply:
Also the service it is planning to replace is /gui/follow not /gui/camera/follow. I'm lazy so the less I have to type the happier I usually am 😅.